nav2_mppi_controller/src/noise_generator.cpp

```diff
@@ -108,14 +168,16 @@ void NoiseGenerator::noiseThread()
 void NoiseGenerator::generateNoisedControls()
 {
   auto & s = settings_;
+  auto ndistribution_vx = std::normal_distribution(0.0f, settings_.sampling_std.vx);
```

Comment:
Is this a "free" or light weight operation? I suppose our previous generator with xtensor definitely wasn't which is why we were doing what we were doing